InfernoRed/module-smartcards
# Feature/election guard trustee

Add support for creating a trustee card.

## To Test
1. ensure your smart card reader is connected and there is a card in it.
2. fire up the api
3. run the `Write Trustee Card` postman request
4. run the `Get Card Long Data` postman request

Expected: the get data returns the same data that is written to the long field in the `Write Trustee Card` request
